docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "How-To: Load Data in ML.NET"
+description: Load Data into ML.NET
+ms.date: 04/01/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to load data from file and streaming data sources.
+---
+
+# How-To: Load Data in ML.NET
+
+This how-to shows you how to load data for processing and training into ML.NET that is stored in files or real-time / streaming data sources.
+
+## Create Data Model Schema
+
+ML.NET allows for data model schemas to be defined via classes. For example, given input data that looks like the following:
+
+```text
+Size (Sq. ft.), HistoricalPrice1 ($), HistoricalPrice2 ($), HistoricalPrice3 ($), Current Price ($)
+700, 100000, 3000000, 250000, 500000
+1000, 600000, 400000, 650000, 700000
+```
+
+You can create a data model that looks like the snippet below:
+
+```csharp
+class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+    [LoadColumn(1,3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+    [LoadColumn(4)]
+    public float CurrentPrice { get; set; }
+}
+```
+
+### Column Attributes
+
+ML.NET provides additional ways to extend data models through attributes.
+
+The `LoadColumn` attribute allows you to specify column indices for your properties. Columns can be loaded individually as is the case of `Size` and `CurrentPrices` or multiple at a time in the form of a vector as is the case of `HistoricalPrices` in the `HousingData` class.
+
+In the case any of your properties is a vector, you can use the `VectorType` attribute in your model. One thing to note however is that all of the elements in the vector need to be of the same type.
+
+## Load Data From A File
+
+When loading data from a file, you can use the `LoadFromTextFile` method along with the data model for the data being loaded and any other parameters. By default, columns are tab-delimited, but the `separatorChar` parameter can be customized to your choosing. Additionally, if the first row of your file is the header, `hasHeader` can be set to `true`.
```

Comment:
How about "To load file data, Use the `LoadFromTextFile` method along with the data model class for the data to be loaded. As the `separatorChar` parameter is tab-delimited by default, change it to your data file separator as needed. If your file has a header, set the `hasHeader` parameter to `true`."